worlds/pokemon_crystal/data/locations.json

```
@@ -7999,17 +8493,19 @@
     "flag": "EVENT_ITEM_FROM_SAGE_GAKU",
     "type": "trainer",
     "tags": [
-      "Trainersanity"
+      "Trainersanity",
+      "Wise Trios Room"
     ],
     "default_item": "NO_ITEM",
     "script": "AP_Trainer_SAGE_GAKU"
   },
   "ITEM_FROM_SAGE_KOJI": {
     "label": "Wise Trios Room - Sage Koji",
     "flag": "EVENT_ITEM_FROM_SAGE_KOJI",
-    "type": "trainer",
+    "type": "giveitem",
```

```suggestion
    "type": "trainer",
```
This will probably crash the game, definitely shouldn't be changed